variables.tf

```diff
@@ -41,10 +41,12 @@ variable "config" {
     firewall = optional(object({
       block_by_default = optional(bool, false)
 
-      aws_managed_rules = optional(set(string), [
-        "AWSManagedRulesAmazonIpReputationList",
-        "AWSManagedRulesCommonRuleSet",
-      ])
+      aws_managed_rules = optional(map(object({
+        rule_action_override = optional(map(string), {})
+      })), {
+        AWSManagedRulesAmazonIpReputationList = {},
+        AWSManagedRulesCommonRuleSet = {}
+      })
```

Comment:
what would a valid rule override then look like?
Could you perhaps give an example in the PR description, and preferably add it to README.md and/or eg. `examples/managed_rule_overrides/main.tf`

Reply:
@nanohest 
I have added some documentation to the README